Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,81 @@
+var humanChoice = "";
+var humanScore = 0;
+var computerScore = 0;
+
+
+$('.choice').click(function() {
```

Comment:
Not three, but just one click handler! (the official term, by the way, for a function that *handles* what occurs upon a given browser event) 💯💯💯 😄